README.md

```diff
@@ -76,7 +76,7 @@ With your own parameters:
     --pix-format            Setting custom pixel/bit format for piping
                             (Default: 'yuv420p10le')
 
-### <center>Segmenting<center>
+<h3 align="center">Segmentation Options</h3>
```

Comment:
Can we change this to "Chunking Options"? "Segmentation" is an actual term for a feature in AV1, so it could become confusing.

Reply:
Sure, no problem.